pytket/extensions/qiskit/qiskit_convert.py

```diff
@@ -747,6 +713,24 @@ def _get_params(
     return [param_to_qiskit(p, symb_map) for p in op.params]
 
 
+def _apply_qiskit_instruction(
```

Comment:
Compare the use of this helper function with the logic in the [original commit](https://github.com/CQCL/pytket-qiskit/pull/475/commits/1929e60be3e076029c099c55d874acb7a36e9567)